libfwupdplugin/fu-common.c

```diff
@@ -305,14 +305,17 @@ fu_error_convert(GError **perror)
 	     FWUPD_ERROR_PERMISSION_DENIED},
 	    {G_USB_DEVICE_ERROR, G_USB_DEVICE_ERROR_BUSY, FWUPD_ERROR_BUSY},
 #endif
+	    {FWUPD_ERROR, FWUPD_ERROR ,FWUPD_ERROR_LAST}
 	};
 
 	/* sanity check */
 	if (error == NULL)
 		return;
 
 	/* find match */
-	for (guint i = 0; i < G_N_ELEMENTS(map); i++) {
+	for (guint i = 0;; i++) {
+		if (map[i].domain == FWUPD_ERROR && map[i].fwupd_code == FWUPD_ERROR_LAST)
+			break;
```

Comment:
can you just wrap this in the #ifdef too? Or drop this commit and review https://github.com/fwupd/fwupd/pull/7412 instead as it deletes the entire section :)

Reply:
Dropped change. Will get into that review later on, thx!